larwirecell/Components/FrameSaver.cxx

```diff
@@ -109,12 +117,23 @@ FrameSaver::configure(const WireCell::Configuration& cfg)
     // geo::kU, geo::kV, geo::kW
     auto wpid = anode->resolve(chid);
     geo::View_t view;
-    switch (wpid.layer()) {
-    case WireCell::kUlayer: view = geo::kU; break;
-    case WireCell::kVlayer: view = geo::kV; break;
-    case WireCell::kWlayer: view = geo::kW; break;
-    default: view = geo::kUnknown;
-    }
+    
+    // switch (wpid.layer()) {
+    // case WireCell::kUlayer: view = geo::kU; break;
+    // case WireCell::kVlayer: view = geo::kV; break;
+    // case WireCell::kWlayer: view = geo::kW; break;
+    // default: view = geo::kUnknown;
+    // }
+    
+    // Use configurable translation between WCT and larsoft 
+    // plane view IDs. Relevant especially for VD 3 view 
+    // since the 2nd induction plane is actually labelled
+    // kY in larsoft vs kV in WCT
+    std::string wct_layer = std::to_string((int)wpid.layer());
+    if (!cfg["plane_map"][wct_layer].isNull())
+      view = (geo::View_t) cfg["plane_map"][wct_layer];
+    else
+      view = geo::kUnknown;
```

Comment:
Can the program meaningfully continue if the view is "unknown"?  In this circumstance, it might be better to simply throw an exception.  I understand this would be changing the behavior wrt previous code, but perhaps it is an appropriate improvement.

Reply:
Removed the guard, I agree allowing the exception is probably the better idea here